workers/java/worker/src/main/java/com/hrznstudio/spatial/worker/entity/EntityWorker.java

```diff
@@ -0,0 +1,79 @@
+package com.hrznstudio.spatial.worker.entity;
+
+import com.hrznstudio.spatial.util.ConnectionManager;
+import com.hrznstudio.spatial.worker.BaseWorker;
+import improbable.Coordinates;
+import improbable.Position;
+import improbable.Vector3f;
+import improbable.collections.Option;
+import improbable.worker.Authority;
+import improbable.worker.EntityId;
+import minecraft.entity.WorldEntity;
+import minecraft.player.PlayerConnection;
+import minecraft.player.PlayerInput;
+import net.minecraft.init.Bootstrap;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class EntityWorker extends BaseWorker.BaseViewWorker {
+
+    public static Map<EntityId, Integer> playerTimeout = new HashMap<>();
```

Comment:
This could probably be handled as an Event or full Component on the player's spatial Entity, without the need to store it in this map.